lib/reporting/proofing_rate_report.rb

```diff
@@ -91,7 +91,7 @@ def reports
             issuers: nil, # all issuers
             time_range: Range.new(
               (end_date - slice_start.days).beginning_of_day,
-              (end_date - slice_end.days).beginning_of_day,
+              (end_date - slice_end.days).end_of_day,
```

Comment:
did we confirm that the three time ranges created are disjoint and have no gaps between them after this change?

Reply:
The sub_reports time ranges are calculated different then the spec file.I see with out updates to the file the time range for three slices as 
```
time_range: 2021-12-02 00:00:00 UTC..2022-01-01 00:00:00 UTC
time_range: 2021-11-02 00:00:00 UTC..2021-12-02 00:00:00 UTC
time_range: 2021-10-03 00:00:00 UTC..2021-11-02 00:00:00 UTC
```
with the updates if we consider end day the time range for for three slices as 
```
2021-12-02 00:00:00 UTC and 2022-01-01 23:59:59 UTC
2021-11-02 00:00:00 UTC and  2021-12-02 23:59:59 UTC
2021-10-03 00:00:00 UTC and 2021-11-02 23:59:59 UTC
```
which is a day overlap. Is this what the exception with end_of_day addition?
or should we consider change the second slice to 
`2021-11-02 00:00:00 UTC and  2021-12-01 23:59:59 UTC
2021-10-03 00:00:00 UTC and 2021-11-01 23:59:59 UTC`